packages/admin/cms-admin/src/blocks/createTextLinkBlock.tsx

```diff
@@ -40,6 +40,20 @@ export function createTextLinkBlock({ link: LinkBlock, name = "TextLink" }: Crea
 
         definesOwnPadding: true,
 
+        extractTextContents: (state) => {
+            return [state.text];
```

Comment:
```suggestion
            return state.text.trim().length > 0 ? [state.text]: [];
```
Should not extract empty strings